.changeset/shiny-lamps-retire.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-llama": patch
+---
+
+Add e2b code artifact tool for the FastAPI template
```

Comment:
e2e test testing the tool?

Reply:
I just added artifact tool to the e2e test (dependency). It now takes about 25 minutes for the tests